api/test/enum.test.js

```diff
@@ -28,9 +28,9 @@ describe('Enum values', () => {
   const testFields = {
     dwellingHouseId: { testValue: 189250 },
     dwellingYearBuilt: { testValue: 1900 },
-    dwellingCity: { testValue: 'Charlottetown' },
-    dwellingRegion: { testValue: 'PE' },
-    dwellingForwardSortationArea: { testValue: 'C1A' },
+    dwellingCity: { testValue: 'Dartmouth' },
```

Comment:
why did you have to change all these javascript files?

Reply:
I missed the warm embrace of javascript